app/models/user.rb

```diff
@@ -2,6 +2,7 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
   has_many :posts, foreign_key: 'author_id'
+  validates :name, presence: true
 
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md) for this project, " `PostsCounter must be an integer greater than or equal to zero.` ", please kindly add validation rule for that. 

Please kindly ensure all custom methods in the models should have a corresponding unit test.